lib/ruby_lsp/ruby_lsp_rspec/definition.rb

```diff
@@ -42,6 +42,7 @@ def on_call_node_enter(node)
         entries.each do |entry|
           # Technically, let can be defined in a different file, but we're not going to handle that case yet
           next unless entry.file_path == @uri.to_standardized_path
+          next unless entry.file_path.end_with?('spec.rb')
```

Comment:
This breaks tests that use tempfiles, do you know how can I make tempfiles use `spec.rb` files? Or do you know any other way to check it's an rspec. I have seen used `node.receiver && node.receiver&.slice != "RSpec"` but it doesn't work always.